packages/smooth_app/lib/database/robotoff_questions_query.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/QueryType.dart';
+import 'package:smooth_app/database/product_query.dart';
+
+class RobotoffQuestionsQuery {
+  RobotoffQuestionsQuery(this.barcode);
+  final String barcode;
+
+  Future<List<RobotoffQuestion>> getRobotoffQuestionsForProduct() async {
+    final RobotoffQuestionResult result =
+        await OpenFoodAPIClient.getRobotoffQuestionsForProduct(
+      barcode,
+      ProductQuery.getLanguage().code,
```

Comment:
cc @M123-dev That's typically something we need to address in off-dart v2.0.0: always use `language` instead of `language.code`

Reply:
@monsieurtanuki https://github.com/openfoodfacts/openfoodfacts-dart/projects/2#card-75976578